### PR TITLE
Add "What is an RO-Crate?" type intro page

### PR DESCRIPTION
**Pull request by elichad**
WIP.

I tried to pull together an overview of RO-Crate for direct users, i.e. with less focus on the RO-Crate _project_ and more focus on the RO-Crate _structure_.

I'll continue working on this next week, feedback welcome in the meantime.

### CODE REVIEW

**Comment by elichad** [comment]
One thing I'm struggling with slightly is how to distinguish between:

- RO-Crate as a singular object, i.e. a collection of data/references with an `ro-crate-metadata.json` file (i.e. the individual user perspective)
- RO-Crate as a project, i.e. its design ethos, use cases, reasons for adoption/integration into existing or new projects/software (i.e. the project leader/infrastructure provider perspective)

and how to design the landing page so that both user groups can easily find the introductory information that's targeted at them

File: about-ro-crate.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+## What is an RO-Crate?
+
+An RO-Crate is a [Research Object](./background.md#research-object-background) (or _RO_) formed of a collection of data (a _crate_) and a special `ro-crate-metadata.json` file which describes the collection.
+
+The collection may contain any kind of research data - papers, data files, software, references to other research, and so on. It may be a folder full of files, an abstract grouping of connected references, or a combination of both.
+
+The `ro-crate-metadata.json` file (also known as the _RO-Crate Metadata Document_) is a plain text file, readable by humans and machines, that includes metadata for each item within the collection - the authors, license, identifier, and so on. Any folder can be turned into an RO-Crate by adding an `ro-crate-metadata.json` file.
+
+Researchers can distribute their work as an RO-Crate to ensure their data travels with its metadata, so that key components are correctly tracked, archived, and attributed. [something about reproducibility]
+
```
Comment (a-mile) [inline about-ro-crate.md:12]: One of the aspects of RO Crate that we find highly valuable is the fact that you get all of the relevant context information for each file (study, publications, contacts, provenance), so I like that you are pointing out that data travel with their metadata, this is key.

File: about-ro-crate.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+* What is the appropriate set of people that should be cited as authors for this dataset?
+
+The person who wants to cite your data will probably not be able to answer these questions accurately and consistently themselves, but you can! Give them all the right information in an `ro-crate-metadata.json` file, and they can cite your software correctly.
+
+Also, if you publish your RO-Crate in an archive or registry, they (or their systems) can re-use the citation metadata from your `ro-crate-metadata.json` file.
+
+## Who is RO-Crate for?
+
+RO-Crate is for everyone! The RO-Crate effort brings together practitioners from very different backgrounds, and with different motivations and use-cases. Among our core target users are:
+
```
Comment (a-mile) [inline about-ro-crate.md:83]: both producers and consumers of data

**Comment by elichad** [comment]
I've finished editing this (CI is expected to fail as not all linked pages are present). Will merge in a few days if there's no further feedback.

**Pull request merged**